.gitlab-ci.yml

```diff
@@ -495,7 +495,7 @@ build_client:
       - $BUILD_CLIENT == "true"
       - $RUN_INTEGRATION_TESTS == "true"
   stage: build
-  dependencies:
+  needs:
     - init_workspace
```

Comment:
Couldn't we add this only one in the `build_and_test_acceptance` template? It was duplicated in all jobs already, but I think it is a good clean-up, once we are on it.

Reply:
Good idea! Implemented and pushed in latest commit.